src/rust/ide/src/controller/module.rs

```diff
@@ -42,7 +53,7 @@ use shapely::shared;
 pub struct Metadata {
     /// Metadata used within ide.
     #[serde(default="default")]
-    pub ide : serde_json::Value,
+    pub ide : IdeMetadata,
```

Comment:
I would name it IDEMetadata

Reply:
I thought we don't use this naming in this repo.

Reply:
I think the recommended one is currently `IdeMetadata`, as Wojciech recently requested uniforming `ID` -> `Id`. Should be confirmed.

Reply:
`IdeMetadata` is correct.